core/src/main/java/haveno/core/trade/statistics/TradeStatistics3.java

```diff
@@ -88,13 +90,27 @@ public static TradeStatistics3 from(Trade trade,
         Offer offer = checkNotNull(trade.getOffer());
         return new TradeStatistics3(offer.getCurrencyCode(),
                 trade.getPrice().getValue(),
-                trade.getAmount().longValueExact(),
+                fuzzTradeAmount(trade.getAmount().longValueExact()),
                 offer.getPaymentMethod().getId(),
-                trade.getTakeOfferDate().getTime(),
+                fuzzTradeDate(trade.getTakeOfferDate().getTime()),
                 truncatedArbitratorNodeAddress,
                 extraDataMap);
     }
 
+    private static long fuzzTradeAmount(long exactAmount) { //  randomize completed trade info #1099
+        long adjustedAmount = (long) ThreadLocalRandom.current().nextDouble(
+                exactAmount * 0.95, exactAmount * 1.05);
+        log.info("fuzzed trade amount from {} to {}", exactAmount, adjustedAmount);
+        return adjustedAmount;
+    }
+
+    private static long fuzzTradeDate(long originalTimestamp) { //  randomize completed trade info #1099
+        long adjustedTimestamp = ThreadLocalRandom.current().nextLong(
+                originalTimestamp-TimeUnit.HOURS.toMillis(24), originalTimestamp);
```

Comment:
This will randomize between the original timestamp and 24 hours prior. Do we want +/- 12 hours instead?

Reply:
True, makes sense.